app/misc/skderivs/skderivs.F90

```diff
@@ -82,13 +82,13 @@ subroutine main(inp)
     write(stdout, "(A)") "Following files will be created:"
     call resize_string(buffer, 1024)
     do ii = 1, size(inp%iHam)
-      fpHam(ii) = getFileId()
+      ! fpHam(ii) = getFileId()
       strTmp = trim(inp%output) // ".ham." // i2c(ii)
       open(fpHam(ii), file=strTmp, status="replace", position="rewind")
```

Comment:
Note, that won't work this way. You have to use `newunit=fpHam(ii)`, if `fpHam(ii)` is not initialized when the `open()` statement is executed!